ui/src/containers/RoleForm/index.js

```diff
@@ -140,7 +141,7 @@ const render = ({ model, handleAttrChange, handleScopeChange }) => {
           id={`${model.get('_id')}nameInput`}
           className="form-control"
           placeholder="A name for this Role"
-          value={model.get('title')}
+          value={model.get('title', '')}
```

Comment:
When `undefined` is passed to value, the input element is regarded as a uncontrolled component, and warning is output into console.

Reply:
Nice good catch 👍